aten/src/ATen/native/native_functions.yaml

```diff
@@ -4686,6 +4686,10 @@
     CPU, NestedTensorCPU: transform_bias_rescale_qkv_cpu
     CUDA, NestedTensorCUDA: transform_bias_rescale_qkv_cuda
 
+- func: _nested_tensor_from_mask(Tensor t, Tensor mask, float min_padding_rate=0) -> Tensor
```

Comment:
What is the idea behind "min_padding_rate"?

Reply:
The reason we have this design is when padding rate is pretty low, which means the tensor is almost not padded, using nested tensor will not have any perf benefit; Instead, it will damage the perf. So we would like to have this while setting it to 0 by default.

Since we actually have a diff calling this nested_tensor_from_mask by default in BetterTransformer, @erichan1 and I both feel it is might be better to keep this functionality. Users can have 0 knowledge of this min_padding_rate if they dont want to use it. Idea?